workflow/log-monitoring/transactions.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Transforming Logs into Transactions
+sidebarTitle: Transactions
+---
+
+Transactions traverse across your system and aggregate your log data with other data that exists outside of the log you are already filtering. This feature is useful when you need to identify broader patterns within your infrastructure that are related to the log you are actively filtering. 
+
+Transactions work by compiling indexing logs based on occurrences of a series of actions, like a user's session or a request handled across your microservice architecture. Examples of transactions range from actions within a mobile app, to function calls and CPU instructions.
+
+# Transaction Measures
+
+Measures are additional ways for you to filter your transactions:
```

Comment:
Measures in this case is for users to add aggregations across the chosen grouping attribute.